src/DI/DeployerExtension.php

```diff
@@ -8,67 +8,95 @@
 use Contributte\Deployer\Runner;
 use Nette\DI\CompilerExtension;
 use Nette\DI\Statement;
+use Nette\Schema\Expect;
+use Nette\Schema\Processor;
+use Nette\Schema\Schema;
 
 /**
  * Deployer Extension
  */
 final class DeployerExtension extends CompilerExtension
 {
 
-	/** @var mixed[] */
-	private $defaults = [
-		'config' => [
-			'mode' => Config::MODE_TEST,
-			'logFile' => '%appDir/../log/deploy.log',
-			'tempDir' => '%appDir/../temp',
-			'colors' => null,
-		],
-		'sections' => [],
-		'userdata' => [],
-		'plugins' => [],
-	];
+    public function getConfigSchema() : Schema
+    {
+        return Expect::structure(
+            [
+            'config' => Expect::structure(
+                [
+                'mode' => Expect::string(Config::MODE_TEST),
+                'logFile' => Expect::string('%appDir/../log/deploy.log'),
+                'tempDir' => Expect::string('%appDir/../temp'),
```

Comment:
Default parameters in schema are not translated to their respective values. They should be added through `$this->getContainerBuilder()->parameters` or `logFile` and `tempDir` should be `->required();`